src/main/java/net/contargo/iris/routedatarevision/api/RouteDataRevisionApiController.java

```diff
@@ -157,12 +158,39 @@ private ResponseEntity<RouteDataRevisionDto> save(RouteDataRevisionDto revision,
                 "routerevision.exists", BAD_REQUEST);
         }
 
+        if (distancesMissmatch(revision)) {
+            throw new RestApiException(messageSource.getMessage("routerevision.countryDistances.missmatch", null,
+                    getLocale()), "routerevision.countryDistances.missmatch", BAD_REQUEST);
```

Comment:
```suggestion
            throw new RestApiException(messageSource.getMessage("routerevision.countryDistances.mismatch", null,
                    getLocale()), "routerevision.countryDistances.mismatch", BAD_REQUEST);
```
I couldn't find the property file where this key is specified. Is it missing?

Reply:
It was there, but was removed by mistake, now it is back again.